docs/diagrams/PlantUML File/DeletePropertySequenceDiagram.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+!include style.puml
+
+hide footbox
+
+participant "Command:CommandDeleteProperty" as CommandDeleteProperty COMMAND_COLOR
+participant "propertyList:PropertyList" as PropertyList PROPERTY_LIST_COLOR
+participant "ui:Ui" as Ui UI_COLOR
+participant "storage:Storage" as Storage STORAGE_COLOR
+participant "pairingList:PairingList" as PairingList PAIRING_LIST_COLOR
+
+->CommandDeleteProperty : execute(Ui, Storage, PropertyList, ClientList, PairingList)
+activate CommandDeleteProperty COMMAND_COLOR
+
+CommandDeleteProperty -> PropertyList : deleteProperty(propertyIndex:int)
+activate PropertyList PROPERTY_LIST_COLOR
+return deletedProperty
```

Comment:
I think the return value should come in the format variable name : variable type

Reply:
Corrected. Good job!